### PR TITLE
Add initial schedule implementation

### PR DESCRIPTION
**Pull request by andrecardoso**
## Description
Add initial schedule implementation. Only duration is considered with simple hide/show transition.

## Motivation and Context
Playlist component schedule 

## How Has This Been Tested?
Added unit tests. Tested locally in the browser.

## Release Plan:
- As the Submitter, upon requesting review of this pull request, I confirm that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed. 
- As the Reviewer, upon approving the changes in this PR, I confirm I have reviewed and I agree that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed

#### Release Checklist Items Skipped?
If any Release Checklist items were intentionally skipped, please provide which ones and the reasons why

### CODE REVIEW

File: demo/src/rise-playlist-dev.html [hunk 1]
```diff
@@ -11,7 +11,11 @@
   <script src="../../node_modules/@webcomponents/webcomponentsjs/custom-elements-es5-adapter.js"></script>
   <script src="https://widgets.risevision.com/beta/common/config-test.min.js"></script>
   <script src="https://widgets.risevision.com/beta/common/common-template.min.js"></script>
+  <script src="https://widgets.risevision.com/beta/common/common-template.min.js"></script>
```
Comment (olegrise) [inline demo/src/rise-playlist-dev.html]: @andrecardoso `common-template` is referenced twice.

File: demo/src/rise-playlist.html [hunk 2]
```diff
@@ -18,7 +18,8 @@
   </script>
   <script src="https://widgets.risevision.com/beta/common/config-test.min.js"></script>
   <script src="https://widgets.risevision.com/beta/common/common-template.min.js"></script>
-  <script src="https://widgets.risevision.com/beta/components/rise-playlist/1/rise-playlist.js"></script>
+  <script src="https://widgets.risevision.com/beta/components/rise-embedded-template/1/rise-embedded-template.js"></script>
+  <script src="https://widgets.risevision.com/beta/components/rise-playlist/2/rise-playlist.js"></script>
```
Comment (olegrise) [inline demo/src/rise-playlist.html]: @andrecardoso should it reference version 2 or 1? Maybe it's a good ides to delete version 2 from GCS.

File: src/schedule.js [hunk 3]
```diff
@@ -9,10 +28,27 @@ export default class Schedule {
   }
 
   stop() {
-
+    clearTimeout(this.itemDurationTimer);
+    this.playingItem = null;
   }
 
   play() {
+    clearTimeout(this.itemDurationTimer);
```
Comment (olegrise) [inline src/schedule.js]: @andrecardoso do you think `play()` should be skipped if `this.items` is empty?

Comment (andrecardoso) [reply src/schedule.js]: Good idea. I'll add a check.

File: demo/src/rise-playlist.html (hunk 2 above)
Comment (andrecardoso) [reply demo/src/rise-playlist.html]: Yes. I'll delete it. Thanks

**Pull request merged**